libs/domain/content/link/link.styles.ts

```diff
@@ -0,0 +1,15 @@
+import { css } from 'lit';
+
+export const styles = css`
+  :host([appearance='dropdown']) {
+    --oryx-button-color: var(--oryx-color-neutral-12);
+
+    padding: 9px 5px;
+    display: flex;
+    height: 24px;
```

Comment:
we should use the max available width, so that you can access/hover anywhere on the real estate of the component.